packages/patterns-react/.env.example

```diff
@@ -4,6 +4,7 @@ PATTERNS_STORYBOOK_SOURCEMAPS=<Boolean to turn on/off sourcemaps in storybook>
 DDS_FLAGS_ALL=<Boolean flag to turn on all feature flags>
 
 DDS_CARD_ARRAY=<Boolean flag to turn on/off the CardArray component>
+DDS_CARD_SECTION=<Boolean flag to turn on/off the CardSection component>
 DDS_CARDS_WITHOUT_IMAGES=<Boolean flag to turn on/off the CardsWithoutImages component>
```

Comment:
Can get rid of this if we have moved to `CardSection`

Reply:
Updated the PR.